content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+﻿---
+title: July '21 Heartbeat
+date: 2021-07-16
+description: |
+  Monthly updates are here! Great new tutorials from the Community, uptick in 
+  jobs requiring DVC, awesome Community discussion on experiments at our June
+  Meetup and a cipher. Can you figure it out?
+descriptionLong: |
+  This month you will find:
+  📈 DVC + Streamlit = ❤️,
+  🇯🇵 DVC in Japanese,
+  📖 A new Udacity Course that includes DVC,
+  🧑🏽‍💻 More and more jobs requiring DVC
+  🧪 June Meetup on Experiments,
+  🚀 New team member, a secret code and more!
+picture: 2021-07-16/july21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/july-heartbeat/825
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - Streamlit
+  - Udacity
+---
+
+# Welcome to Summer!
+
+![It's summer!](https://media.giphy.com/media/WuY9yfI89DbNu/giphy.gif)
+
+# From the Community
+
+<span style="color:purple">A</span>s usual we have a ton of goodness from the
+Community! Let's jump in!
+
+## Antoine Toubhans Post Combining Streamlit and DVC!
+
+[Antoine Toubhans](https://www.linkedin.com/in/antoine-toubhans-92262119/) of
+[Sicara](https://www.sicara.fr/) wrote a fantastic and detailed tutorial
+bringing together the best of DVC and integrating it with Streamlit to provide a
+customizable UI. The tutorial goes through the steps of setting up a pipeline,
+spltting a dataset, training and evaluating a model, tracking changes to data
+and model, dvc <span style="color:purple">m</span>etrics and plots and then
+bridging the gap in visualizations using Streamlit. You won't want to miss this
+one!
+
+<external-link
+href="https://www.sicara.ai/blog/dvc-streamlit-webui-ml"
+title="How to Build Customizable Web UI for Machine Learning with Streamlit and DVC"
+description="Antoine Toubhans shares a detailed tutorial on using DVC with Streamlit to provide a customizable UI"
+link="https://www.sicara.ai/"
+image="/uploads/images/2021-07-16/streamlit.png"/>
+
+https://media.giphy.com/media/l0MYGtVSzAqKfkcEg/giphy.gif
+
+## DVC and CML in Japanese!
+
+For our friends that speak Japanese,
+[these slides](https://www.slideshare.net/yusukeshibui/testing-machine-learningdevelopment)
+created by
+[Yusuke Shibui](https://www.slideshare.net/yusukeshibui?utm_campaign=profiletracking&utm_medium=sssite&utm_source=ssslideview)
+walk you through a <span style="color:purple">m</span>achine learning to
+production project using DVC and CML. We love seeing our tools being used all
+around the world! 🌏
+
+![](/uploads/images/2021-07-16/in-japanese.png)
```

Comment:
add a subtitle with a source?

Reply:
@shcheklein I'm unsure of what you mean here "with a source."  I've looked back through old posts to see how source would be included in subtitle, but not seeing it.  Can you clarify?

Reply:
yes, sure, sorry that it was not clear:

here are some examples:

https://dvc.org/blog/ml-best-practices-in-pytorch-dev-conf-2018 (head image)
https://dvc.org/blog/gsod-ideas-2020 (images with subtitles with links in them)
https://dvc.org/blog/best-practices-of-orchestrating-python-and-r-code-in-ml-projects (also head image)

idea here is that we should utilize the subtitles capabilities where it's possible (it makes it easier to read).

here are some examples btw of what you can do with images:

https://dvc.org/doc/user-guide/contributing/blog#adding-images

e.g. wrap text like:

`![](/uploads/images/2020-02-10/image.png 'Some Title :wrap-left =500')`

also, it's a good practice to put some description into the `[]` - it works good for SEO, for people who have slower connection, etc


^^^ all these things are not blockers in any way to make it clear :)

Reply:
Ah!  Ok.  Got it now.  Is the `[]` after the `!` what holds Alt text?

I've corrected!